api/registerOrganisation/index.ts

```diff
@@ -108,3 +110,17 @@ function convertEmptyStringToNull(term: string): string {
   return term === '' ? null : term;
 }
 
+function securePbaNumberErrorMessage(data, errorMessage: string): string {
+  const pbaNumberErrorMessage = '6 : PBA_NUMBER Invalid or already exists';
+  const emailDuplicateErrorMessage = '6 : EMAIL Invalid or already exists';
+
+  if (errorMessage.toLowerCase() === pbaNumberErrorMessage.toLowerCase()) {
+    return 'Registration cannot be completed';
+  }
+
+  if (errorMessage.toLowerCase() === emailDuplicateErrorMessage.toLowerCase()) {
+    return 'A user with this email address already exists. You should check that the email address has been entered correctly. You should also check if your organisation has already been registered. If you\'re still having problems, contact HMCTS.';
+  }
+
```

Comment:
Can you check in here if errorDescription is null , and if so return a default message

Reply:
I think you need to make sure that no stack trace is ever returned. What happens if the backend returns some other error other than the 2 you are handling

Reply:
I have updated it with a generic message.